script.mjs

```diff
@@ -30,9 +30,10 @@ window.read_saved_url = function (callback) {
     new Promise(async (resolve) => {
         try {
             const url = new URL(location);
-            const urlsafe = url.searchParams.get("saved");
             url.search = "";
             history.replaceState(null, "", url);
+
+            const urlsafe = url.searchParams.get("saved");
```

Comment:
With this reordering the value is always `null`, isn't it? Because the search was wiped out before.

Reply:
Arf, forgot to revert. I was testing a few things out.